core/arch/arm/kernel/generic_boot.c

```diff
@@ -760,21 +760,25 @@ static uint64_t get_dt_val_and_advance(const void *data, size_t *offs,
 static int add_res_mem_dt_node(struct dt_descriptor *dt, const char *name,
 			       paddr_t pa, size_t size)
 {
-	int offs;
-	int ret;
-	int addr_size = 2;
-	int len_size = 2;
+	int offs = 0;
+	int ret = 0;
+	int addr_size = -1;
+	int len_size = -1;
 	char subnode_name[80];
```

Comment:
could you init this one also? ` = { 0 };`

Reply:
sure.